Gruntfile.js

```diff
@@ -17,20 +17,18 @@ module.exports = function (grunt) {
     grunt.initConfig({
         // Project package file destination.
         pkg: grunt.file.readJSON("package.json"),
-        eslint: {
-            all: ["**/*.js"]
-        },
-        jsonlint: {
-            all: ["*.json", ".*.json", "tests/*.json"]
+        lintAll: {
+            sources: {
+                md: [ "**/*.md", "!node_modules/**/*.md"],
+                js: ["**/*.js"],
+                json: ["*.json", ".*.json", "tests/*.json", "!package-lock.json"],
+                other: ["./.*", "!node_modules/**/*.*", "!./**/.DS_Store"]
```

Comment:
What files is this rule intended to catch? It seems more reasonable to just list them individually or in groups.

Reply:
Other is for trailing linespaces.
https://www.npmjs.com/package/gpii-grunt-lint-all

Reply:
Yes - I expect it makes more sense to just list LICENCE, .gitignore and .eslintignore explicitly than this convoluted pattern. What if someone started to check images, etc. into the repo?